Sources/Sentry/SentryNetworkTrackingIntegration.m

```diff
@@ -2,21 +2,19 @@
 #import "SentryLog.h"
 #import "SentryNSURLSessionTaskSearch.h"
 #import "SentryNetworkTracker.h"
-#import "SentryOptions+Private.h"
 #import "SentryOptions.h"
 #import "SentrySwizzle.h"
 #import <objc/runtime.h>
 
 @implementation SentryNetworkTrackingIntegration
 
-- (void)installWithOptions:(SentryOptions *)options
+- (BOOL)installWithOptions:(SentryOptions *)options
 {
     if (!options.enableSwizzling) {
         [SentryLog logWithMessage:
                        @"Not going to enable NetworkTracking because enableSwizzling is disabled."
                          andLevel:kSentryLevelDebug];
-        [options removeEnabledIntegration:NSStringFromClass([self class])];
-        return;
+        return NO;
```

Comment:
I think we could use all options to enable the networkTracking for this integration, and manually check for the enableNetworkBreadcrumbs.

Reply:
SentryNetworkTrackingIntegration is a complicated one, but I made some improvements.